### PR TITLE
chore: prepare release v2.8.1

### PR DESCRIPTION
**Pull request by JorTurFer**
Signed-off-by: Jorge Turrado <jorge_turrado@hotmail.es>

_Provide a description of what has been changed_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))

Relates to https://github.com/kedacore/keda/issues/3626

### CODE REVIEW

**Comment by tomkerkhove** [comment]
Relates to https://github.com/kedacore/keda/issues/3626

File: CHANGELOG.md [hunk 1]
```diff
@@ -33,29 +34,33 @@ To learn more about our roadmap, we recommend reading [this document](ROADMAP.md
...
-- TODO ([#XXX](https://github.com/kedacore/keda/issues/XXX))
+- TODO ([#XXX](https://github.com/kedacore/keda/issue/XXX))
+
+### Other
+
+- TODO ([#XXX](https://github.com/kedacore/keda/issue/XXX))
+
+## v2.8.1
+
+### Fixes
```
Comment (tomkerkhove) [inline CHANGELOG.md:59]: Can you please add the sections that have no entries and use "None" to be consistent please?

Comment (JorTurFer) [reply CHANGELOG.md:59]: sure

Comment (JorTurFer) [reply CHANGELOG.md:63]: done

**Review by zroubalik — APPROVED** [review]

**Comment by tomkerkhove** [comment]
@JorTurFer You forgot to add the features section :) but it's OK.

**Comment by tomkerkhove** [comment]
image

But have opened a PR - https://github.com/kedacore/keda/pull/3628

**Comment by JorTurFer** [comment]
f**k! Nice catch

**Pull request merged**